interactive-visualizers/scripts/upload_gcp.sh

```diff
@@ -15,18 +15,21 @@
 
 set -e
 
-# get package directory
-PACKAGE_DIR="dist/interactive-visualizers/*"
+# get packages directories
+VISUALIZER_PACKAGE_DIR="dist/interactive-visualizers/*"
+PLAYGROUND_PACKAGE_DIR="dist/playground/*"
 # host url
 PACKAGE_HOST="interactive_visualizer"
 
 rm -rf dist/
 yarn prod-build
+yarn playground-prod-build
 
 PACKAGE_VERSION=`node -p "require('./package.json').version"`
 echo 'current version: ' $PACKAGE_VERSION
 # remove the pre-built addon tarball if it already exist
 if [ "$1" = "for-publish" ]; then
   echo 'copying ...'
-  gsutil -m cp $PACKAGE_DIR gs://$PACKAGE_HOST/$PACKAGE_VERSION
+  gsutil -m cp $VISUALIZER_PACKAGE_DIR gs://$PACKAGE_HOST/$PACKAGE_VERSION
+  gsutil -m cp $PLAYGROUND_PACKAGE_DIR gs://$PACKAGE_HOST/$PACKAGE_VERSION/playground
```

Comment:
Does the playground app require the colocation of the visualizer package? or does it compile the visualizer app within its dist directory?

Reply:
The 2 projects are codebase-wise independent. The playground app compiles on its own. It only depends on the interactive visualizers at runtime (by iframing it).
I'm thinking though that for deployment it could be good to keep both of them under a same version directory, which would ensure compatibility.
What are your thoughts on this?